test/e2e/beat/recipes_test.go

```diff
@@ -16,7 +17,9 @@ import (
 	beatcommon "github.com/elastic/cloud-on-k8s/pkg/controller/beat/common"
 	"github.com/elastic/cloud-on-k8s/pkg/controller/common/settings"
 	"github.com/elastic/cloud-on-k8s/pkg/controller/common/version"
+	essettings "github.com/elastic/cloud-on-k8s/pkg/controller/elasticsearch/settings"
```

Comment:
nit: it's a bit strange to import this package here to grab `$POD_IP`, which is not related to es settings in that context. I think I would maybe re-declare the env var here.